src/man/sss-certmap.5.xml

```diff
@@ -44,7 +44,9 @@
         <para>
             The rules are processed by priority while the number '0' (zero)
             indicates the highest priority. The higher the number the lower is
-            the priority. A missing value indicates the lowest priority.
+	    the priority. A missing value indicates the lowest priority. The
+	    processing is stopped when a matched rule is found and no further
+	    rules are checked.
```

Comment:
@amitkumar50, please, do not use tabs in place where we're using space indentation. It has to be fixed in the lines you have changed/added.